README.md

```diff
@@ -39,78 +39,95 @@
 </p>
 </br>
 
-Semgrep is a fast, open-source, static analysis engine for finding bugs, detecting vulnerabilities in third-party dependencies, and enforcing code standards. Semgrep analyzes code locally on your computer or in your build environment: **code is never uploaded**. [Get started →.](#getting-started-)
+Semgrep is a fast static analysis engine designed to detect vulnerabilities, find bugs, and enforce code standards. Semgrep analyzes code locally on your computer or in your build environment: **by default, code is never uploaded**. [Get started →.](#getting-started-)
 
 <a href="#option-1-getting-started-from-the-cli">
 <img src="https://raw.githubusercontent.com/returntocorp/semgrep/develop/images/semgrep-scan-cli.jpg" alt="Semgrep CLI image"/></a>
 
 ### Language support
 
-Semgrep supports 30+ languages.
+Semgrep Code supports over 30+ languages.
 
 | Category     | Languages                                                                                                                                                                     |
 | ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | GA           | C# · Go · Java · JavaScript · JSX · JSON · PHP · Python · Ruby · Scala · Terraform · TypeScript · TSX                                                                         |
 | Beta         | Kotlin · Rust                                                                                                                                                                 |
 | Experimental | Bash · C · C++ · Clojure · Dart · Dockerfile · Elixir · HTML · Julia · Jsonnet · Lisp · Lua · OCaml · R · Scheme · Solidity · Swift · YAML · XML · Generic (ERB, Jinja, etc.) |
 
-### Getting started 🚀
 
-1. [From the CLI](#option-1-getting-started-from-the-cli)
-2. [From the Semgrep Cloud Platform](#option-2-getting-started-from-the-semgrep-cloud-platform-recommended)
+Semgrep Supply Chain supports 8 languages across over 15 package managers. 
```

Comment:
`over 30+ languages` and `over 15 package managers` doesn't seem very consistent.

I think we should drop the `+` for both or use the `+` and drop the `over`